.github/workflows/pr_tests.yml

```diff
@@ -0,0 +1,39 @@
+name: Run non-slow tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+env:
+  HF_HOME: /mnt/cache
+  OMP_NUM_THREADS: 8
+  MKL_NUM_THREADS: 8
+  PYTEST_TIMEOUT: 60
+
+jobs:
+  run_tests_single_gpu:
```

Comment:
Ok to just test on GPU for me now, but we'll soon need a CPU-only version (we'll add Flax very soon). Happy to do this in a follow-up PR though